includes/Checker/Checks/Plugin_Repo/File_Type_Check.php

```diff
@@ -304,7 +382,7 @@ function ( $file ) use ( $plugin_path ) {
 	 * @return string Description.
 	 */
 	public function get_description(): string {
-		return __( 'Detects the usage of hidden and compressed files, VCS directories, application files and badly named files.', 'plugin-check' );
+		return __( 'Detects the usage of hidden and compressed files, VCS directories, application files, badly named files and library core files.', 'plugin-check' );
```

Comment:
```suggestion
		return __( 'Detects the usage of hidden and compressed files, VCS directories, application files, badly named files and core library files.', 'plugin-check' );
```

Not a native speaker but this sounds more correct

Reply:
All naming is based on that: `Library Core Files`. Should we rename those accordingly?